Content.Shared/Psionics/PsionicPowerPrototype.cs

```diff
@@ -0,0 +1,61 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Psionics;
+
+[Prototype("psionicPower")]
+public sealed partial class PsionicPowerPrototype : IPrototype
+{
+    /// <summary>
+    ///     The ID of the psionic power to use.
+    /// </summary>
+    [IdDataField]
+    public string ID { get; } = default!;
+
+    /// <summary>
+    ///     The name of the psionic power.
+    /// </summary>
+    [DataField(required: true)]
+    public string Name = default!;
+
+    /// <summary>
+    ///     The description of a power in yml, used for player notifications.
+    /// </summary>
+    [DataField(required: true)]
+    public string Description = default!;
+
+    /// <summary>
+    ///     The list of each Action that this power adds in the form of ActionId and ActionEntity
+    /// </summary>
+    [DataField]
+    public List<EntProtoId> Actions = new();
+
+    /// <summary>
+    ///     The list of what Components this power adds.
+    /// </summary>
+    [DataField]
+    public readonly List<string>? Components = new();
```

Comment:
`ComponentRegistry` should be the Datatype for a list of Components. Please don't just use Strings.